publish_wc.py

```diff
@@ -0,0 +1,166 @@
+
+#!/usr/bin/python
```

Comment:
The bash comment needs to be at the first line

```suggestion
#!/usr/bin/python
```